README.md

```diff
@@ -44,9 +44,11 @@ This file contains a number of front-end interview questions that can be used wh
 * If you could master one technology this year, what would it be?
 * What are the differences between Long-Polling, Websockets and SSE?
 * Explain the importance of standards and standards bodies.
-* What is FOUC? How do you avoid FOUC?
+* What is Flash of Unstyled Content (FOUC)? How do you avoid FOUC?
 * Do your best to describe the process from the time you type in a website's URL to it finishing loading on your screen.
-* Explain what ARIA and screenreaders are, and how to make a website accessible.
+* What is your approach to accessiblity and developing in keyboard accessibility and visual focus, such as using foundational HTML and Javascript techniques? 
+* Explain your experience with Web Accessibility Initiative's Accessible Rich Internet Applications (WAI-ARIA): Do you know the difference between developing an aceesible website following Web Content Accessibility Guidelines (WCAG) web standards and custom developing for assistive technologies such as screenreaders with WAI-ARIA?
```

Comment:
Well, I thought of that, but then I thought some of the interviewers might
not have a solid grasp on the topics and thus expanded the formal names
instead of using the informal or acronym only. So I'll vote no for now :)

On Tue, Jan 20, 2015 at 3:41 PM, Rich Gilbank <notifications@github.com>
wrote:

> In README.md
> <https://github.com/h5bp/Front-end-Developer-Interview-Questions/pull/226#discussion-diff-23253744>
> :
>
> >  * Do your best to describe the process from the time you type in a website's URL to it finishing loading on your screen.
> > -* Explain what ARIA and screenreaders are, and how to make a website accessible.
> > +* What is your approach to accessiblity and developing in keyboard accessibility and visual focus, such as using foundational HTML and Javascript techniques?
> > +* Explain your experience with Web Accessibility Initiative's Accessible Rich Internet Applications (WAI-ARIA): Do you know the difference between developing an aceesible website following Web Content Accessibility Guidelines (WCAG) web standards and custom developing for assistive technologies such as screenreaders with WAI-ARIA?
>
> Mind rewording this a bit? This could probably be shortened and simplified.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/h5bp/Front-end-Developer-Interview-Questions/pull/226/files#r23253744>
> .
>



-- 
(703) 942 9569 google voice 24x7
"You have access to the greatest source of knowledge in the universe."
"Well, I do talk to myself sometimes." ~ The Invasion of Time (1978)